tests/test_model_variant_name_manager.py

```diff
@@ -17,188 +17,109 @@
 import unittest
 from unittest.mock import patch
 
+from tests.common.test_utils import default_encode
+from model_analyzer.constants import DEFAULT_CONFIG_PARAMS
+
 
 class TestModelVariantNameManager(trc.TestResultCollector):
```

Comment:
Add a test that has more complicated dict matching than a single string-int dict.  I'd like to see multiple dicts inside of a list, and then have the list reordered on a 2nd try (but still be the same)

Reply:
I can create a test with nested dicts. But, lists aren't going to work if they're reordered, as they wouldn't be the same.  The actual model_config does have lists, but the order they are in is always consistent.

Reply:
Done